rust_src/src/windows.rs

```diff
@@ -165,9 +170,45 @@ pub fn window_margins(window: LispObject) -> LispObject {
     )
 }
 
+/// Return combination limit of window WINDOW.
+/// WINDOW must be a valid window used in horizontal or vertical combination.
+/// If the return value is nil, child windows of WINDOW can be recombined with
+/// WINDOW's siblings.  A return value of t means that child windows of
+/// WINDOW are never (re-)combined with WINDOW's siblings.
+#[lisp_fn]
+pub fn window_combination_limit(window: LispObject) -> LispObject {
+    let w = window.as_window_or_error();
+
+    if !w.is_internal() {
+        error!("Combination limit is meaningful for internal windows only");
+    }
+
+    LispObject::from_raw(w.combination_limit)
+}
+
+/// Set combination limit of window WINDOW to LIMIT; return LIMIT.
+/// WINDOW must be a valid window used in horizontal or vertical combination.
+/// If LIMIT is nil, child windows of WINDOW can be recombined with WINDOW's
+/// siblings.  LIMIT t means that child windows of WINDOW are never
+/// \(re-)combined with WINDOW's siblings.  Other values are reserved for
+/// future use.
+
 /// Return the window which was selected when entering the minibuffer.
 /// Returns nil, if selected window is not a minibuffer window.
 #[lisp_fn]
+pub fn set_window_combination_limit(window: LispObject, limit: LispObject) -> LispObject {
+    let mut w = window.as_window_or_error();
+
+    if !w.is_internal() {
+        error!("Combination limit is meaningful for internal windows only");
+    }
+
+    w.combination_limit = limit.to_raw();
+
+    limit
+}
+
```

Comment:
docstring for minibuffer_selected_window is misplaced

Reply:
Looks like a merge mixup. Good catch.